### PR TITLE
Animation

### PR DESCRIPTION
**Pull request by ezrasong**
Added animation to the blog title and renamed the file so it routes properly

### CODE REVIEW

File: src/components/Blog/Blog.css [hunk 1]
```diff
@@ -35,3 +35,24 @@
...
+  width: 0;
+  height: 0.15em;
+  background-color: rgb(62, 62, 62);
+  transition: width 0.3s ease-in-out;
+}
+
+.animate-underline:hover::before {
+  width: 100%;
+}
+/*Pain*/
```
Comment (jpabadir) [inline src/components/Blog/Blog.css:58]: ?

**Review by jpabadir — CHANGES_REQUESTED** [review]
Looks good but can you make it so that it does it anytime any of the thing is hovered? The whole clickable area should be what triggers this.

**Comment by ezrasong** [comment]
> Looks good but can you make it so that it does it anytime any of the thing is hovered? The whole clickable area should be what triggers this.

so the image as well?

**Comment by ezrasong** [comment]
Alright changed it

**Pull request merged**